code/style.css

```diff
@@ -1,4 +1,308 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: rgb(204, 204, 204);
+  color: rgb(0, 0, 0);
+  font-weight: normal;
+}
+
+body{
+  margin: 0px;
+  font-family: 'Raleway', sans-serif;
+}
+
+/*Header and navbar*/
+
+.flex-container {
+  height:75px;
+  background-color:darkgrey;
+  color:black;
+  font-size: 25px;
+  display:flex;
```

Comment:
Regarding the requirements: You used flexbox for all sections and to my eye, everything seems to work fine (only part to fix for that: `display: block`, see other comment). And your design is responsive as asked, I tried it on my phone (not only Chrome inspector).